src/modules/m_namedmodes.cpp

```diff
@@ -185,7 +185,7 @@ class ModuleNamedModes : public Module
 					}
 
 					// Change parameter to the text after the '='
-					curr.param = value;
+					curr.param = std::move(value);
```

Comment:
The v3 branch has to build as GNU++03 so `std::move` can't be used. This can be committed to master though.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4